kotlin/src/main/com/looker/sdk/4.0/models.kt

```diff
@@ -41,16 +42,17 @@ data class AccessToken (
   /**
    * Access Token used for API calls (read-only)
    */
+  @param:JsonProperty("access_token") @get:JsonProperty("access_token")
```

Comment:
why did this show up? seems like `name == safeName(name)`

Reply:
whoops. that was some test code to verify imports and behavior. It will disappear when I regenerate. Good catch!